src/ray/rpc/grpc_server.cc

```diff
@@ -40,6 +40,7 @@ GrpcServer::GrpcServer(std::string name,
       is_closed_(true),
       num_threads_(num_threads),
       keepalive_time_ms_(keepalive_time_ms) {
+  RAY_CHECK(num_threads_ > 0) << "Num of threads in gRPC must be greater than 0";
```

Comment:
seems like a good case for an unsigned type 😄

Reply:
Yes, but potentially it still could be 0 if it's unsigned type.